src/cms/static/css/style.less

```diff
@@ -344,13 +344,9 @@ h2.heading {
     }
 }
 
-input[type=file].image-field {
-    width: 0.1px;
-	height: 0.1px;
-	opacity: 0;
-	overflow: hidden;
-	position: absolute;
-    z-index: -1;
+.image-field {
+    display: none;
+    position: absolute;
 }
```

Comment:
I don't understand here the author's purpose. Setting such small width/height and opacity to 0 is just like hiding the element so I reduced this code. I kept on hiding it, because that Image Field Widget (see [https://github.com/django/django/blob/master/django/forms/templates/django/forms/widgets/clearable_file_input.html](url) can not be customized with css in accordance to the form styles.

Reply:
But hiding the elements means also, there is no more a checkbox (included in the widget) for deleting the image. If you like to, I could add a customized button for deleting the icon.

Reply:
I don't get this width/height parameters either... So yes, `display:none` should be the way to go. Just a hint: you can directly assign the [tailwind class](https://tailwindcss.com/docs/display#class-reference) `hidden` instead of `image-field`.

Yes, the checkbox to remove the icon would be cool. You don't have to leave the design as it is, If you wanted to style the widget yourself, you can always define a custom widget which inherits from the default one in the form class:
```
class IconWidget(ClearableFileInput):
    template_name = "/path/to/custom/widget/template"

class RegionForm(forms.ModelForm):
    ...
    class Meta:
        ...
            "icon",
        ],
        widgets = {
            "icon": IconWidget,
        }
```

Reply:
Ah, and one general note regarding your GitHub comments: I noticed that your links always point to https://github.com/Integreat/cms-django/pull/url.
So probably you're using the markdown feature incorrecly. it's `[link-text](url)` but you tend to fill your urls in the first part where the link text should go and leave the `url` part unchanged...
Apart from that: If you just want to paste the url without a different link text, you don't need special markdown syntax for this. just insert the url and GitHub will automatically convert it to a link.

Reply:
Thank you, I just fixed the link. Now it should work.

Reply:
You're proposed IconWidget sounds good for me. I will check that.

Reply:
Yes, works now. And apart from the remove-checkbox: I think it would also be cool to see the filename anywhere after a file has been selected. This way you can be sure the selection worked before submitting the form.